### ISSUE TITLE
WAI-ARIA Technology Notes: Old UA descriptions

### ISSUE THREAD

**Issue by momdo** [w3c/wcag#148]
In <a href="https://www.w3.org/WAI/GL/2016/WD-WCAG20-TECHS-20160105/wai-aria_notes.html#wai-aria_ua_support">User Agent Support for WAI-ARIA</a>:

>- Firefox 1.5 and Firefox 2.0 partially supports WAI-ARIA, however it requires the use of namespaces, and doesn't support the use of Liveregions.
>- Firefox 3+ contains better support for WAI-ARIA, including Liveregions.
>- IE8 partially supports WAI-ARIA.
>- JAWS 8 and Window-Eyes 5.5+ partially support WAI-ARIA.
>- Jaws 10+ supports WAI-ARIA.
>- FireVox, a self-voicing extension to Firefox, also supports WAI-ARIA via direct DOM access.
>- NVDA partially supports WAI-ARIA.

These UA explanations are too old. At least, referring to Firefox 1.5 and Firefox 2.0 is unnecessary. <a href="https://blogs.msdn.microsoft.com/ie/2014/08/07/stay-up-to-date-with-internet-explorer/">IE8 is out-of-date</a>.

**Comment by awkawk** [comment]
If you have suggestions for specific changes you think we should make, please provide those in a comment or pull request.

**Comment by joshueoconnor** [comment]
It is hard to make blanket assertions about user agent and browser support. We can say some aspects of a language are supported where others are not. Support for role information has in general improved but there are issues with controls like TAB menus . State and property information can be patchy - aria-owns/aria-controls

//Draft User Agent update 2016

        *Firefox 43 has 'good' working support for WAI-ARIA.
        *IE11+ has 'some' support but there are issues (such as restricting ARIA to the use of what Microsoft term 'accessible elements' that affects aria-labelledby/aria-describedby).
        *JAWS 17 has 'good' support for WAI-ARIA. 
        *NVDA has 'good' support for WAI-ARIA (especially when used with in combination with new builds of Firefox).
@@ add more info

A useful ARIA/Screen reader support matrix can be found at Power Mapper. http://www.powermapper.com/tests/screen-readers/aria/index.html

**Comment by joshueoconnor** [comment]
Surveyed Jan 22nd https://www.w3.org/2002/09/wbs/35422/Jan26_2016/results

**Comment by momdo** [comment]
I have consulted <a href="http://waic.jp/">WAIC</a> for an advice for this issue. WAIC is an organization responsible for the Japanese translation of WCAG 2.0.

The answer was that:
1) WAIC recognizes the importance to provide information concerning UA/AT supporting the techniques, from the view point of Accessibility Supported. However, it agrees that the current WAI-ARIA is unclear. Especially considering that the depth of support of the techniques depends on the region it is used, e.g. depending on language used, the aim of the draft is unclear and WAIC doubts the need for such ambiguous and outdated information.
2) WAIC does not have any effective measure to test UA/AT and publish informative articles regarding the support of WAI-ARIA by them.

Namely, WAIC did not propose any concreate alternative for this issue. Though, I got a suggestion that it is worth considering to remove all those uninformative information.

By the way, WAIC provides accessibility supported information in Japanese. (http://waic.jp/docs/jis2010/as.html)
WAIC has a plan to update accessibility supported test cases, but it says it does not have test faces for WAI-ARIA techniques at this time.
